src/otx/engine/engine.py

```diff
@@ -479,7 +479,12 @@ def predict(
             if explain_config is None:
                 explain_config = ExplainConfig()
 
-            predict_result = process_saliency_maps_in_pred_entity(predict_result, explain_config)
+            for transform in datamodule.config.test_subset.transforms:
+                tranf_name = transform["class_path"].split(".")[-1]
+                if tranf_name == "Resize" and transform["init_args"].get("keep_ratio", False):
+                    explain_config.crop_padded_map = True
```

Comment:
Does the "keep_ratio=True" mean that we need to crop & pad the saliency map? Is it general for all tasks?

Reply:
Yes, but based on my investigation it's true only for YOLOX-based models